src/Jackett.Common/Indexers/Abstract/GazelleTracker.cs

```diff
@@ -217,17 +225,12 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 throw new Exception(errorReason);
             }
 
-
             try
             {
                 var json = JObject.Parse(response.ContentString);
                 foreach (JObject r in json["response"]["results"])
                 {
-                    // groupTime may be a unixTime or a datetime string
-                    var isNumber = long.TryParse((string)r["groupTime"], out long n);
-                    var groupTime = (isNumber)
-                        ? DateTimeUtil.UnixTimestampToDateTime(long.Parse((string)r["groupTime"]))
-                        : DateTimeUtil.FromFuzzyTime((string)r["groupTime"]);
+                    var groupTime = DateTimeUtil.UnixTimestampToDateTime(long.Parse((string)r["groupTime"]));
```

Comment:
OK. did some debugging and discovered that orpheus was failing with `your api key did not work` 
but not because the apikey was actually invalid, or not picked up from the config correctly,
but because the subsequent test to perform a keyworldless query as part of the config validation was failing to process the returned json of the search results.
my log shows that the error was at line 233 while processing the grouptime.

After I restored the previous lines 226-230 to replace 233 my indexer started working again.

Please restore that grouptime logic

Reply:
Restored. Probably some leftover code from my migration. It might've not been affecting all results that are why in my testing of Orpheus, It worked for me.

Nevertheless, Made the changes and Tested them on Orpheus and GGN, With and Without Query + Downloads.